coremltools/converters/mil/frontend/torch/test/test_executorch_e2e.py

```diff
@@ -92,6 +94,139 @@ def forward(self, a, x, b):
             backend=("mlprogram", "fp16"),
         )
 
+    def test_softmax(self):
+        class LinearModule(torch.nn.Module):
+            def __init__(self):
+                super().__init__()
+                self.softmax = torch.nn.Softmax()
+
+            def forward(self, x):
+                z = self.softmax(x)
+                return z
+
+        model = LinearModule()
+        model.eval()
+
+        self.run_compare_torch(
+            [(2, 2)], model, frontend=TorchFrontend.EDGEIR, backend=("mlprogram", "fp16")
+        )
+
+    @pytest.mark.xfail(reason="numerical error")
```

Comment:
How big is the numerical error here? Maybe you should use the `atol` or `rtol` parameter to `self.run_compare_torch`.

Reply:
A bit too big to `atol` and `rtol`...

For deeplab_v3
```
E           AssertionError: 
E           Not equal to tolerance rtol=0.05, atol=0.5
E           
E           Mismatched elements: 34 / 1053696 (0.00323%)
E           Max absolute difference: 0.97145367
E           Max relative difference: 24053.578
E            x: array([[[[ 7.101562,  7.101562,  7.101562, ...,  6.992188,  6.992188,
E                      6.992188],
E                    [ 7.101562,  7.101562,  7.101562, ...,  6.992188,  6.992188,...
E            y: array([[[[ 7.097789,  7.097789,  7.097789, ...,  7.022431,  7.022431,
E                      7.022431],
E                    [ 7.097789,  7.097789,  7.097789, ...,  7.022431,  7.022431,...
```
For enformer_join
```
E           AssertionError: 
E           Not equal to tolerance rtol=0.05, atol=0.5
E           
E           Mismatched elements: 67 / 128 (52.3%)
E           Max absolute difference: 1.89292
E           Max relative difference: 106.33904
E            x: array([ 0.519165,  0.754537, -0.61137 , -0.046065, -0.706869, -0.791332,
E                  -0.261786,  0.253299, -0.893944, -0.101306, -0.437432, -0.509161,
E                  -0.167917, -0.420541, -0.767622,  0.033906,  0.839066,  0.024934,...
E            y: array([ 0.687215,  0.315739, -0.684043,  0.538311, -0.086759, -0.007733,
E                  -0.648528, -0.867819, -0.821678, -0.542535,  0.718882,  0.836924,
E                   0.825016,  0.572487, -0.343973, -0.426525, -0.255493, -0.370776,...
```